db/dcrpg/queries.go

```diff
@@ -308,9 +308,12 @@ func InsertTickets(db *sql.DB, dbTxns []*dbtypes.Tx, txDbIDs []uint64, checked,
 // subsequent cache lookups by other consumers will succeed.
 //
 // Outputs are slices of DB row IDs for the votes and misses, and an error.
+// votesMilestones holds upto date blockchain info deployment data needed
+// to update changes in agendas table.
```

Comment:
Sorry to be a stickler, but the reason I brought up this bit of docs is because the docs for InsertVotes didn't mention that it updated the agendas table too.  Note that it describes it's updates to votes and misses.

So, this added comment is good, but you've not yet described what's being updated in the agendas table or that InsertVotes is now responsible for keeping the agendas table current.